src/pudl/package_data/eia191/column_maps/data.csv

```diff
@@ -0,0 +1,18 @@
+year_index,any_year
+year,year
+month,month
+report_state,report_state
+id,id
+gas_field_code,gas_field_code
+reservoir_code,reservoir_code
+company_name,company_name
+field_name,field_name
+reservoir_name,reservoir_name
+field_type,field_type
+county_name,county_name
+status,status
+base_gas,base_gas
+working_gas_capacity_(mcf),working_gas_capacity_
```

Comment:
Should be changed as we actually drop the parentheses before we rename the columns.

```suggestion
working_gas_capacity_mcf,working_gas_capacity_mcf
total_field_capacity_mcf,total_field_capacity_mcf
maximum_daily_delivery_mcf,maximum_daily_delivery_mcf
```